website/src/components/SearchPage/DownloadDialog/DownloadDialog.spec.tsx

```diff
@@ -91,21 +102,29 @@ describe('DownloadDialog', () => {
         });
         await checkAgreement();
 
-        expect(getDownloadHref()).toBe(
-            `${defaultLapisUrl}/sample/details?downloadAsFile=true&versionStatus=LATEST_VERSION&isRevocation=false&dataUseTerms=OPEN&dataFormat=tsv&accessionVersion=SEQID1&accessionVersion=SEQID2`,
+        let [path, query] = getDownloadHref()?.split('?') ?? [];
+        expect(path).toBe(`${defaultLapisUrl}/sample/details`);
+        expect(query).toMatch(
+            /downloadAsFile=true&downloadFileBasename=Loculus_ebola_metadata_\d\d\d\d-\d\d-\d\d-\d\d\d\d&versionStatus=LATEST_VERSION&isRevocation=false&dataUseTerms=OPEN&dataFormat=tsv&accessionVersion=SEQID1&accessionVersion=SEQID2/,
         );
 
         await userEvent.click(screen.getByLabelText(/Yes, include older versions/));
         await userEvent.click(screen.getByLabelText(/Raw nucleotide sequences/));
         await userEvent.click(screen.getByLabelText(/Gzip/));
-        expect(getDownloadHref()).toBe(
-            `${defaultLapisUrl}/sample/unalignedNucleotideSequences?downloadAsFile=true&dataUseTerms=OPEN&compression=gzip&accessionVersion=SEQID1&accessionVersion=SEQID2`,
+
+        [path, query] = getDownloadHref()?.split('?') ?? [];
+        expect(path).toBe(`${defaultLapisUrl}/sample/unalignedNucleotideSequences`);
+        expect(query).toMatch(
+            /downloadAsFile=true&downloadFileBasename=Loculus_ebola_unalignedNucleotideSequences_\d\d\d\d-\d\d-\d\d-\d\d\d\d&dataUseTerms=OPEN&compression=gzip&accessionVersion=SEQID1&accessionVersion=SEQID2/,
```

Comment:
```suggestion
            /downloadAsFile=true&downloadFileBasename=loculus_ebola_unaligned-nucleotide-sequences_\d\d\d\d-\d\d-\d\d-\d\d\d\d&dataUseTerms=OPEN&compression=gzip&accessionVersion=SEQID1&accessionVersion=SEQID2/,
```

cleaner file name if there's no casing irregularity

Reply:
tests adapted: https://github.com/loculus-project/loculus/pull/3021/commits/4e18a6718aab31b2e885246fa06de4b2ba127f8c